src/platform_impl/linux/wayland/state.rs

```diff
@@ -307,7 +307,11 @@ impl WindowHandler for WinitState {
                 &mut self.events_sink,
             );
 
-        self.window_compositor_updates[pos].size = Some(new_size);
+        // NOTE: Only update when the value is `Some` to not override consequent configures with
+        // the same sizes.
+        if new_size.is_some() {
+            self.window_compositor_updates[pos].size = new_size;
```

Comment:
I think the comment is pretty clear that

`Some -> None` shouldn't make `None`, because we could have 2 configures and one will be `Some`, but the following `None` because it'll be the same as the one before.

Reply:
Fair enough, I thought it would always have been consumed already.

Reply:
Then the comment shouldn't say: `to not override consequent configures with the same sizes` because there is no size, it was `None` (the fact that it was `None` maybe because the size didn't change, isn't common knowledge here?). Maybe it should say something along these lines:

```rust
// new_size will be `None` when the size didn't change, but a previous resize might still be pending so we only override it when it is `Some`
```

Reply:
yeah, applied.